### ISSUE TITLE
Add file uploads

### ISSUE THREAD

**Issue by DiptoChakrabarty** [DiptoChakrabarty/node_app#1]
Currently the only way images are being displayed is by providing  an image url to the application , we should also have a method to upload images from our system through the backend .

**Comment by iathul** [comment]
@DiptoChakrabarty What about using multer for uploading images

**Comment by DiptoChakrabarty** [comment]
> @DiptoChakrabarty What about using multer for uploading images

yes @iathul  that would work , I just need a way to upload images and displaying them

**Comment by iathul** [comment]
Then let's work on it

**Comment by insaurabh** [comment]
@iathul Are you working on it, else I can work.

**Comment by iathul** [comment]
@insaurabh  you can work on it

**Comment by RajatSablok** [comment]
@DiptoChakrabarty I can work on this, can you brief me about what you want exactly?

**Comment by DiptoChakrabarty** [comment]
@RajatSablok  right now after signup/signin if a user wants to create a new post he has to enter three details in a form , the post name , description and image . 
Right now it is coded such that I can only put an image url and that image is displayed , I am looking for a way to upload  images from my local system and that should be displayed later on .
